scripts/scp_api.py

```diff
@@ -298,14 +315,16 @@ def get_studies(self, dry_run=False):
         '''
 
         print("GET STUDIES")
-        resp = self.do_get(self.api + "studies", dry_run=dry_run)
+        resp = self.do_get(self.api_base + "studies", dry_run=dry_run)
         if dry_run:
             print("DRY_RUN:: Returned dummy names.")
             resp[c_STUDIES_RET_KEY] = ["DRY_RUN 1", "DRY_RUN 2"]
         else:
             if(resp[c_SUCCESS_RET_KEY]):
-                self.studies = [str(element['name']) for element in resp[c_RESPONSE].json()]
-                self.name_to_id = [[str(element['name']), str(element['_id']['$oid'])] for element in resp[c_RESPONSE].json()]
+                studies_json = resp[c_RESPONSE].json()
+                self.study_objects = studies_json
+                self.studies = [str(element['name']) for element in studies_json]
```

Comment:
It might be nice to be able to use either the name of the study or the accession.